src/contracts/arbitrum/CrosschainForwarderArbitrum.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/console.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {AaveV3Arbitrum} from 'aave-address-book/AaveV3Arbitrum.sol';
+import {IInbox} from 'governance-crosschain-bridges/contracts/dependencies/arbitrum/interfaces/IInbox.sol';
+import {IL2BridgeExecutor} from 'governance-crosschain-bridges/contracts/interfaces/IL2BridgeExecutor.sol';
+
+/**
+ * @title A generic executor for proposals targeting the arbitrum v3 pool
+ * @author BGD Labs
+ * @notice You can **only** use this executor when the arbitrum payload has a `execute()` signature without parameters
+ * @notice You can **only** use this executor when the arbitrum payload is expected to be executed via `DELEGATECALL`
+ * @notice This contract assumes to be called via AAVE Governance V2
+ * @notice This contract will assume the SHORT_EXECUTOR will be topped up with enough funds to fund the short executor
+ * @dev This executor is a generic wrapper to be used with Arbitrum Inbox (https://developer.offchainlabs.com/arbos/l1-to-l2-messaging)
+ * It encodes a parameterless `execute()` with delegate calls and a specified target.
+ * This encoded abi is then send to the Inbox to be synced executed on the arbitrum network.
+ * Once synced the ARBITRUM_BRIDGE_EXECUTOR will queue the execution of the payload.
+ */
+contract CrosschainForwarderArbitrum {
+  IInbox public constant INBOX =
+    IInbox(0x4Dbd4fc535Ac27206064B68FfCf827b0A60BAB3f);
+  address public constant ARBITRUM_BRIDGE_EXECUTOR =
+    0x7d9103572bE58FfE99dc390E8246f02dcAe6f611;
+  address public constant ARBITRUM_GUARDIAN =
+    0xbbd9f90699c1FA0D7A65870D241DD1f1217c96Eb;
+
+  // amount of gwei to overpay on basefee for fast submission
+  uint256 public constant BASE_FEE_MARGIN = 10 gwei;
```

Comment:
@miguelmtzinf in the end didn't got with *2, as it seems to excessive.
In theory we don't need to overpay at all, so overpaying 10gwei should always be sufficient.

Reply:
Agree